ec2/spark_ec2.py

```diff
@@ -145,6 +145,14 @@ def parse_args():
         default=DEFAULT_SPARK_GITHUB_REPO,
         help="Github repo from which to checkout supplied commit hash (default: %default)")
     parser.add_option(
+        "--spark-ec2-git-repo",
+        default=DEFAULT_SPARK_EC2_GITHUB_REPO,
+        help="Github repo from which to checkout spark-ec2 (default: %default)")
+    parser.add_option(
+        "--spark-ec2-branch",
```

Comment:
Nit: This should probably be `--spark-ec2-git-branch` to match `--spark-ec2-git-repo`.

Or alternately, `--spark-ec2-repo` and `--spark-ec2-branch`, since we don't take any git repo (it has to be on GitHub) and we call that out in the help text.